docs/root/intro/version_history.rst

```diff
@@ -14,6 +14,7 @@ Version history
 * circuit-breaker: added cx_open, rq_pending_open, rq_open and rq_retry_open gauges to expose live
   state via :ref:`circuit breakers statistics <config_cluster_manager_cluster_stats_circuit_breakers>`.
 * cluster: set a default of 1s for :ref:`option <envoy_api_field_Cluster.CommonLbConfig.update_merge_window>`.
+* config: removed deprecated_v1 bind_to_port option.
```

Comment:
Do you think we should link to the `lds.proto` file here?

Reply:
I don't think it's necessary but I'm happy to add a link in my next PR (since I'm deprecating SDS in my next PR, and keeping this PR up to date has been a challenge)